odiglet/pkg/instrumentation/instrumentlang/python.go

```diff
@@ -1,58 +1,13 @@
 package instrumentlang
 
 import (
-	"fmt"
-
 	"github.com/odigos-io/odigos/common"
-	"github.com/odigos-io/odigos/odiglet/pkg/env"
-	"github.com/odigos-io/odigos/odiglet/pkg/instrumentation/consts"
 	"k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
 )
 
-const (
-	envOtelTracesExporter              = "OTEL_TRACES_EXPORTER"
-	envOtelMetricsExporter             = "OTEL_METRICS_EXPORTER"
-	envOtelLogsExporter                = "OTEL_LOGS_EXPORTER"
-	envLogCorrelation                  = "OTEL_PYTHON_LOG_CORRELATION"
-	envOtelExporterOTLPTracesProtocol  = "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"
-	pythonConfiguratorEnvVar           = "OTEL_PYTHON_CONFIGURATOR"
-	pythonConfiguratorValue            = "odigos-python-configurator"
-	envOtelExporterOTLPMetricsProtocol = "OTEL_EXPORTER_OTLP_METRICS_PROTOCOL"
-	httpProtobufProtocol               = "http/protobuf"
-	pythonOdigosOpampServer            = "ODIGOS_OPAMP_SERVER_HOST"
-	pythonOdigosDeviceId               = "ODIGOS_INSTRUMENTATION_DEVICE_ID"
-)
-
 func Python(deviceId string, uniqueDestinationSignals map[common.ObservabilitySignal]struct{}) *v1beta1.ContainerAllocateResponse {
-	otlpEndpoint := fmt.Sprintf("http://%s:%d", env.Current.NodeIP, consts.OTLPHttpPort)
-	opampServerHost := fmt.Sprintf("%s:%d", env.Current.NodeIP, consts.OpAMPPort)
-
-	logsExporter := "none"
-	metricsExporter := "none"
-	tracesExporter := "none"
-
-	if _, ok := uniqueDestinationSignals[common.MetricsObservabilitySignal]; ok {
-		metricsExporter = "otlp"
-	}
-	if _, ok := uniqueDestinationSignals[common.TracesObservabilitySignal]; ok {
-		tracesExporter = "otlp"
-	}
 
 	return &v1beta1.ContainerAllocateResponse{
-		Envs: map[string]string{
```

Comment:
please keep the envs for the device until we sunset it.
So that we don't need to think about edge race conditions while migrating, and can quickly revert back to using the device if needed.